src/main/scala/org/broadinstitute/dsde/workbench/sam/model/SamModel.scala

```diff
@@ -104,5 +107,6 @@ case class AccessPolicyResponseEntry(policyName: AccessPolicyName, policy: Acces
 case class BasicWorkbenchGroup(id: WorkbenchGroupName, members: Set[WorkbenchSubject], email: WorkbenchEmail) extends WorkbenchGroup
 
 case class ManagedGroupMembershipEntry(groupName: ResourceId, role: AccessPolicyName, groupEmail: WorkbenchEmail)
+case class ManagedGroupAccessInstructions(groupName: String, instructions: String)
```

Comment:
why is groupName in here? Can this be
`case class ManagedGroupAccessInstructions(value: String) extend ValueObject`